docker-compose.yml

```diff
@@ -1,17 +1,31 @@
-web:
-  build: .
-  dockerfile: Dockerfile
-  ports:
-    - 8000:8000
-  links:
-    - redis
-  environment:
-    - PYTHONUNBUFFERED=1
-    - PYTHONASYNCIODEBUG=1
-    - GENE_TO_REACTIONS_API
-    - GENE_TO_REACTIONS_PORT
-    - ID_MAPPER_API
-redis:
-  image: redis
-  ports:
-    - 6379
+version: "2.1"
+services:
+  web:
+    build:
+      context: .
+      dockerfile: Dockerfile
+    ports:
+      - 8000:8000
+    links:
+      - redis
+    environment:
+      - PYTHONUNBUFFERED=1
+      - PYTHONASYNCIODEBUG=1
+      - GENE_TO_REACTIONS_API=${GENE_TO_REACTIONS_API:-139.59.134.152}
```

Comment:
Yes, but the service is then also changing a bit so to be handled in different pr